e2e/cypress/integration/scroll/editing_deleting_scroll_spec.js

```diff
@@ -0,0 +1,103 @@
+        });
+    });
+
+    it('MM-T2371, MM-T2372 Post list does not scroll when the offscreen post is edited or deleted', () => {
+        // # Other user posts a multiline message
+        cy.postMessageAs({sender: otherUser, message: multilineString, channelId: testChannelId});
+        cy.getLastPostId().then((postId) => {
+            const multilineMessageID = postId;
+
+            // # Other user posts a few messages so that the multiline message is hidden
+            Cypress._.times(30, (postIndex) => {
+                cy.postMessageAs({sender: otherUser, message: `Other users p-${postIndex}`, channelId: testChannelId});
+            });
+
+            // # Scroll above the last few messages
+            cy.get('div.post-list__dynamic', {timeout: TIMEOUTS.ONE_SEC}).should('be.visible').
+                scrollTo(0, '90%', {duration: TIMEOUTS.ONE_SEC}).
+                wait(TIMEOUTS.ONE_SEC);
+
+            // # Get the text of the first visible post
+            cy.get('.post-message__text:visible').first().then((postMessage) => {
+                firstPostBeforeScroll = postMessage.text();
+            });
+
+            // # Get the text of the last visible post
+            cy.get('.post-message__text:visible').last().then((postMessage) => {
+                lastPostBeforeScroll = postMessage.text();
+            });
+
+            // # Amend multiline message from the other user
+            cy.externalRequest({user: otherUser, method: 'PUT', path: `posts/${multilineMessageID}`, data: {id: multilineMessageID, message: newMultilineMessage}});
+
+            // # Wait for the message to be amended
+            cy.wait(TIMEOUTS.ONE_SEC);
+
+            // * Verify the first post is the same after deleting
+            cy.get('.post-message__text:visible').first().then((firstPostAfterScroll) => {
+                expect(firstPostAfterScroll.text()).equal(firstPostBeforeScroll);
+            });
+
+            // * Verify the last post is the same after deleting
+            cy.get('.post-message__text:visible').last().then((lastPostAfterScroll) => {
+                expect(lastPostAfterScroll.text()).equal(lastPostBeforeScroll);
+            });
+
+            // # Remove multiline message from the other user
+            cy.externalRequest({user: otherUser, method: 'DELETE', path: `posts/${multilineMessageID}`});
+
+            // # Wait for the message to be deleted
+            cy.wait(TIMEOUTS.ONE_SEC);
+
+            // * Verify the first post is the same after deleting
+            cy.get('.post-message__text:visible').first().then((firstPostAfterScroll) => {
+                expect(firstPostAfterScroll.text()).equal(firstPostBeforeScroll);
+            });
+
+            // * Verify the last post is the same after deleting
+            cy.get('.post-message__text:visible').last().then((lastPostAfterScroll) => {
+                expect(lastPostAfterScroll.text()).equal(lastPostBeforeScroll);
```

Comment:
Since we're only verifying the effect of editing on scrolling in this test and the test for deleting posts by other users were covered in your [other PR](https://github.com/mattermost/mattermost-webapp/pull/8176), I don't think we'll need to repeat this test here again, so it's safe to remove this.

Reply:
Hello,
Thanks for your review, @prapti !
I've changed the comments, as you've suggested. 
Also, I've noticed that there was some confusion over the scope of changes. It was caused by the fact that my local repo wasn't synced to the remote one. As the result, it looked like I created a whole new file, while in fact, I've only added a few more checks to the existing scenario. I've fetched the updates, and now the diff should look correctly
So as for the rest, I believe no changes are required.

A bit more details below:
As I've mentioned in the [original message](https://github.com/mattermost/mattermost-webapp/pull/8203#issue-661285568), this pull request modifies the [previous one](https://github.com/mattermost/mattermost-webapp/pull/8176), adding few more checks to the scenario. This corresponds to the initial descriptions of the test cases:
- Editing - https://automation-test-cases.vercel.app/test/MM-T2371
- Deleting - https://automation-test-cases.vercel.app/test/MM-T2372
Namely, the 'Deleting' test case says: "delete the multi-line post edited above". Consequently, I feel like these 2 test cases should be together.

I've renamed the original file and expanded it, so there's no code duplication.

@prapti Could you please have another look?
Thanks!

Reply:
We only deal with one test case per block. If you wish to have two test cases, please separate them into two different blocks with two different titles. Thanks

Reply:
But to Prapti's point, you already covered the delete part on another PR, so no need to duplicate it here.